projecttacoma/fqm-testify
# Add trust meta profile option

# Summary
- This PR adds a UI component that allows the user to specify whether or not they want to run calculation with `trustMetaProfile`. `trustMetaProfile` is an option in `fqm-execution` and is defaulted to false. 

## New Behavior
- The user can now opt to set `trustMetaProfile` to true through the UI.

## Code Changes
- `README.md` - added short sentence about `trustMetaProfile` (is this enough?)
- `PopulationCalculation.tsx` - set `calculateSDEs` to `true` and `trustMetaProfile` to the value of the `trustMetaProfile` atom in the CalculationOptions.
- `PatientCreationPanel.tsx`/`ResourceDisplay.tsx`/`ResourcePanel.tsx`/`generate-test-cases.tsx` - pass `trustMetaProfile` atom value into calls to `calculateDetailedResult()`
- `MeasureFileUploadHeader.tsx` - moved popover to be next to the "Step" for consistency
- `package.json`/`package-lock.json` - upgraded fqm-execution version.
- `index.tsx` - Added switch component for `trustMetaProfile`
- `trustMetaProfile.ts` - created an atom for `trustMetaProfile`
- `MeasureCalculation.ts` - added `trustMetaProfile` as a parameter to `calculateDetailedResult()`

# Testing Guidance
- `npm run check`
- `npm run dev`
- Navigate to `localhost:3000`
- Upload `CMS1028` measure bundle (one example) and make sure the `useTrustMetaProfile` switch is off.
- Upload the associated test cases and click "Calculate Population Results". An error from cql-execution should pop up.
- Repeat these steps but set `useTrustMetaProfile` to yes. There should no longer be an error from cql-execution.
- Do we like the switch component for this? Would it make more sense to have a check box? Any other suggestions?
